### ISSUE TITLE
Right align problem

### ISSUE THREAD

**Issue by derekperkins** [justjanne/powerline-go#114]
I'm trying out `-eval -right-modules` from #102 but having some issues where rendering looks fine, but the output includes the env var names and includes extra quotation marks.

I'm running oh-my-zsh in iTerm2 on a Macbook Pro on 10.14.2.

![image](https://user-images.githubusercontent.com/3588778/51882158-31f9ae00-233b-11e9-8265-976271317fdb.png)

### .zshrc

```
zmodload zsh/datetime

function preexec() {
  __TIMER=$EPOCHREALTIME
}

function powerline_precmd() {
  local __ERRCODE=$?
  local __DURATION=0

  if [ -n $__TIMER ]; then
    local __ERT=$EPOCHREALTIME
    __DURATION="$(($__ERT - ${__TIMER:-__ERT}))"
  fi

  PS1="$(powerline-go -eval -modules nix-shell,venv,user,host,ssh,cwd,perms,git,hg,jobs,exit,root,vgo -modules-right docker,duration,kube -duration $__DURATION -error $__ERRCODE -shell zsh -newline -path-aliases \$GOPATH/src/github.com=@GOPATH-GH -shorten-gke-names -numeric-exit-codes)"
  unset __TIMER
}

function install_powerline_precmd() {
  for s in "${precmd_functions[@]}"; do
    if [ "$s" = "powerline_precmd" ]; then
      return
    fi
  done
  precmd_functions+=(powerline_precmd)
}

if [ "$TERM" != "linux" ]; then
    install_powerline_precmd
fi
```

**Comment by john-stone** [comment]
Same issue on Manjaro Linux 4.20.3-1, zsh 5.6.2.

**Comment by tillyboy** [comment]
Instead of using `PS1="$(powerline-go...)"` in your config, try `eval "$(powerline-go...)"` That fixed it for me (Manjaro 4.19.24-1, zsh 5.7.1)
This fix is also suggested at the bottom of the `README.md` file.

**Comment by pdf** [comment]
Sorry for the confusion - this feature was merged before I'd added documentation, I'll send a PR to clean up the current docs.  For posterity, I use the following in a system-wide include (in `/etc/profile.d`) to support both bash and zsh with the same module-set:

```bash
POWERLINE_BIN="/usr/local/bin/powerline-go"

if [ "$TERM" != "linux" -a -f "$POWERLINE_BIN" ]; then
        POWERLINE_FONT_MODE="patched"
        POWERLINE_MODULES_LEFT="user,cwd,perms,jobs"
        POWERLINE_MODULES_RIGHT="exit,duration,git,vgo,docker,kube"
        if [ -n "${SSH_CLIENT}" ] ; then
                POWERLINE_MODULES_LEFT="host,user,cwd,perms,jobs"
        fi

        POWERLINE_CMD="${POWERLINE_BIN} -shorten-gke-names -modules ${POWERLINE_MODULES_LEFT} -modules-right ${POWERLINE_MODULES_RIGHT} -colorize-hostname -numeric-exit-codes -eval"

        if [ -n "${BASH_VERSION:-}" ] ; then
                INTERACTIVE_BASHPID_TIMER="/tmp/${USER}.START.$$"
                PS0='$(echo $SECONDS > "$INTERACTIVE_BASHPID_TIMER")'

                function _update_ps1() {
                        local __ERRCODE=$?
                        local __DURATION=0

                        if [ -r $INTERACTIVE_BASHPID_TIMER ]; then
                                local __END=$SECONDS
                                local __START=$(cat "$INTERACTIVE_BASHPID_TIMER")
                                __DURATION="$(($__END - ${__START:-__END}))"
                                rm -f "$INTERACTIVE_BASHPID_TIMER"
                        fi

                        eval "$(${POWERLINE_CMD} -shell bash -mode $POWERLINE_FONT_MODE -duration $__DURATION -error $__ERRCODE)"

                }

                PROMPT_COMMAND="_update_ps1"

        elif [ -n "${ZSH_VERSION:-}" ] ; then
                zmodload zsh/datetime

                function preexec() {
                        __TIMER=$EPOCHREALTIME
                }

                function powerline_precmd() {
                        local __ERRCODE=$?
                        local __DURATION=0

                        if [ -n $__TIMER ] ; then
                                local __ERT=$EPOCHREALTIME
                                __DURATION="$(($__ERT - ${__TIMER:-__ERT}))"
                        fi

                        eval "$(${=POWERLINE_CMD} -shell zsh -mode $POWERLINE_FONT_MODE -duration $__DURATION -error $__ERRCODE)"
                }

                function install_powerline_precmd() {
                  for s in "${precmd_functions[@]}"; do
                        if [ "$s" = "powerline_precmd" ]; then
                          return
                        fi
                  done
                  precmd_functions+=(powerline_precmd)
                }

                install_powerline_precmd
        fi
fi
```
